tvtk/tests/test_tvtk.py

```diff
@@ -200,7 +200,7 @@ def test_object_cache(self):
         # The test sometimes fails as VTK seems to generate objects with the
         # same memory address and hash, we try to force it to allocate more
         # objects so as to not end up reusing the same address and hash.
-        junk = [vtk.vtkConeSource() for i in range(5)]
+        junk = [tvtk.ConeSource() for i in range(50)]
```

Comment:
Reading the comment before the specific line of code looks that there is a bug somewhere between the tvk and tvtk layer. Is there a related bug issue for this problem?